utilities/EluvioLiveCli.js

```diff
@@ -3342,6 +3342,7 @@ yargs(hideBin(process.argv))
   )
 
   .strict()
+  .wrap(yargs().terminalWidth() != null ? Math.min(120, yargs().terminalWidth()) : 120)
```

Comment:
need the null check to deal w/ `elv-live | less`, and the min just feels better for extremely wide terminals